src/components/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -3,28 +3,28 @@ import React from "react"
 import ContentLoader from "react-content-loader"
 
 export interface SkeletonLoaderProps {
-  variant?: string
-  width?: string
-  height?: string
+  variant?: "circle" | "text"
```

Comment:
S: I think we should change the variants to `"circle" | "rect"` to be more semantic.
For instance, we can use the rect variant to inputs.